module/os/linux/zfs/zpl_file.c

```diff
@@ -27,6 +27,7 @@
 #ifdef CONFIG_COMPAT
 #include <linux/compat.h>
 #endif
+#include <sys/fcntl.h>
```

Comment:
A little surprisingly `linux/fs.h` doesn't appear to already be included here.  It looks like adding it should resolve the arm64 build failure.

Reply:
Yes, it should be <linux/fs.h>